### PR TITLE
Sliders

### PR DESCRIPTION
**Pull request by trietpham252**

### CODE REVIEW

**Comment by trietpham252** [comment]
Screenshot 2023-12-19 at 22 51 34

File: admin/src/pages/dashboard/index.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+        <Col span={6} style={{ fontSize: '30px' }}>
+          <BellOutlined />
+        </Col>
+      </Row>
+      <SearchBar
+        prefix={<SearchOutlined style={{ fontSize: 20 }} />}
+        placeholder='Search for...'
+        style={styleSearch}
+      />
+      <ButtonSlider
```
Comment (llamas-laptrinh) [inline admin/src/pages/dashboard/index.tsx]: render using  array.map()

File: admin/src/pages/dashboard/index.tsx
Comment (llamas-laptrinh) [inline admin/src/pages/dashboard/index.tsx:1]: update nested routes

**Comment by trietpham252** [comment]
Screenshot 2023-12-22 at 19 30 25
Screenshot 2023-12-22 at 19 30 32
ab851bcf176">
Screenshot 2023-12-22 at 19 30 38

**Pull request merged**